libs/pika/execution/tests/unit/algorithm_let_error.cpp

```diff
@@ -229,7 +229,7 @@ int main()
         PIKA_TEST(!let_error_callback_called);
     }
 
-    test_adl_isolation(ex::let_error(ex::just(), my_namespace::my_type{}));
+    test_adl_isolation(ex::let_error(ex::just_error(42), my_namespace::my_let_error_type{}));
```

Comment:
To do: we don't have `just_error` and `just_done` in pika's `std::execution` implementation.

Reply:
I updated this to use our internal `error_sender` intended for testing. I'll hold off adding `just_done` and `just_error` as an incentive to prefer `PIKA_WITH_STDEXEC=ON`.